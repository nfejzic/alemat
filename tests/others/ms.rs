use alemat::{elements::StrLiteral, MathMl, MathMlFormatter};

#[test]
fn str_lit() {
    let out =
        MathMl::with_content(StrLiteral::from("Hello there")).render_with(&mut MathMlFormatter);

    crate::snap_test!(out, name: "others_str_literal");
}
