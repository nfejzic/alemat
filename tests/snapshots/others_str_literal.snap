---
source: tests/others/ms.rs
expression: input
---
<math>
  <ms>
    Hello there
  </ms>
</math>

